### ISSUE TITLE
actions-on-google not available in Maven Central

### ISSUE THREAD

**Issue by steveperkins** [Ticketmaster/actions-on-google-kotlin#33]
Has this artifact been moved or renamed? README.md shows this POM dependency:
```
<dependency>
	<groupId>com.tmsdurham.actions</groupId>
	<artifactId>actions-on-google</artifactId>
	<version>1.5.0</version>
	<type>pom</type>
</dependency>
```

but [mvnrepository.com can't find it](https://mvnrepository.com/search?q=actions-on-google). 
[Maven Central search](https://search.maven.org/#search%7Cga%7C1%7Cactions-on-google) returns only one result that doesn't match the README:
```
Group ID: ca.sukhsingh.actions (README says  'com.tmsdurham.actions')
Artifact ID: actions-on-google (matches README)
Latest Version: 1.0.2 (README says 1.5.0)
```

Is there a new group ID or artifact ID?

**Comment by ggrell** [comment]
@steveperkins I just checked https://github.com/Ticketmaster/actions-on-google-kotlin/blob/master/sdk/build.gradle and it looks like the group id changed to: `com.tmsdurham.actions`
and visible here on jcenter (not Maven Central): https://bintray.com/patjackson52/maven/com.tmsdurham.actions

**Comment by steveperkins** [comment]
The README for this project shows that it's intended to be in Maven Central as well under group ID com.tmsdurham.actions. Is the README wrong? Is the package in Maven Central under a different name?
![actions-on-google-kotlin](https://user-images.githubusercontent.com/2751150/33796833-2c437fc4-dcc2-11e7-8d14-c13bfbea7a36.PNG)

**Comment by patjackson52** [comment]
Sorry, about that misleading readme.  I am in process of getting it on maven.  It is hosted on JCenter for now.  I do not use the maven build tool.  If the documentation is missing something to configure maven to use jcenter, a PR to update the docs would be appreciated.

**Comment by davidmarinangeli** [comment]
Hi Patrick, I'm experiencing the same issue. I've added jcenter repository on my pom through
```  
<repositories>
        <repository>
            <id>jcenter</id>
            <url>https://jcenter.bintray.com/</url>
        </repository>
    </repositories>
```

but it still gives me error: the only way I can use it is through IntelliJ search menu and still it gives me the 1.2.7 as last version (the others simply give me error). 

Have you fixed this somehow?

**Comment by davidmarinangeli** [comment]
Hi Patrick, I'm using jcenter repo to add your library on my project. Unfortunately it doesn't import DialogflowApp object (com.tmrdurham.actions.DialogflowApp). 

Is it a common problem? Do you know some fix?

**Comment by patjackson52** [comment]
@davidmarinangeli are you using maven?  I have not used maven with this artifact and I am not much help in that area.

**Comment by davidmarinangeli** [comment]
Yep, Maven: nevermind, I will try to make that work or I will create a gradle project. Have a nice weekend :)

**Comment by morbyx** [comment]
Indeed "actions-on-google" looks pretty empty. 
Try using "actions-on-google-core" instead. That one seems to have all the objects.
I've tested it today with Maven with the JCenter repository added.

**Comment by patjackson52** [comment]
@morbyx yes, -core is the bulk of the lib.  actions-on-google has a dependency on it, so both may be needed.  Were you able to get going?
Also V2 is in the works, so this may change slightly in the near future if you want V2.

**Comment by davidmarinangeli** [comment]
I'm waiting for V2, these days I'm working with the node.js library but I would be really happy to come back to Kotlin/Java ;)
